hangman.js

```diff
@@ -1 +1,107 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var word = prompt('gimme a word:');
+var guesses = [];
+var attempts = word.length + 4;
+var attemptsLeft = attempts;
+var display = '';
+
+myl = 'a';
+place = word.indexOf(myl);
+
+function showDisplay() {
+  for(i = 0; i < word.length; i++) {
+    display += '-';
+  }
+  display;
+}
+function getGuess() {
+  if(display === word) {
+    attemptsLeft = 0;
+    win();
+  } else {
+    while (attemptsLeft > 0) {
+      var guess = prompt('You have ' + attemptsLeft + ' attempts. ' + 'guess a letter:\n' + display);
+      var place = word.indexOf(guess);
+      if (place > -1) {
+          attemptsLeft--;
+          display = display.replace(display[place], guess);
+          getGuess();
+
+      } else if (place < 0) {
+        attemptsLeft--;
+        if(attemtps == 0) {
```

Comment:
When I try to play, the game stops and gives me an error: `Uncaught ReferenceError: attemtps is not defined(…)` in my console, which is saying "hey! I don't understand this word/variable, it hasn't been defined". In this case it's just a spelling error on line 32, but definitely use your console for help with debugging!